react-webpack/.mocharc.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  reporter: 'spec',
+  slow: 75,
+  timeout: 2000,
+  spec: 'tests/**/*.spec.js',
```

Comment:
Again with the nonstd test directory?

Reply:
Think trying to demonstrate too many things at once. Will remove.